src/main/scala/scala/tools/refactoring/implementations/oimports/TreeToolbox.scala

```diff
@@ -0,0 +1,52 @@
+package scala.tools.refactoring
+package implementations.oimports
+
```

Comment:
That's interesting: If I organize imports in this file, the implementation adds
```scala
import scala.collection.mutable
```
here.